index.d.ts

```diff
@@ -1,6 +1,7 @@
 import { ClientRequest, IncomingMessage, ServerResponse } from "http";
 import * as opentracing from "opentracing";
 import { SpanOptions } from "opentracing/lib/tracer";
+import { TransportRequestParams } from "@elastic/elasticsearch/lib/Transport"
```

Comment:
I just realized that this introduces a hard dependency on the library. I guess there is no other option than to copy the definition as you mentioned earlier. Sorry about the oversight.

Reply:
No problem! I just took out the package and followed the pattern for the `EventEmitter` interface when adding the interface type.